libfreerdp/core/freerdp.c

```diff
@@ -627,7 +627,11 @@ UINT32 freerdp_error_info(freerdp* instance)
 	return instance->context->rdp->errorInfo;
 }
 
-void freerdp_set_error_info(rdpRdp* rdp, UINT32 error) {
+void freerdp_set_error_info(rdpRdp* rdp, UINT32 error)
+{
+	if (!rdp)
+		return;
```

Comment:
freerdp_set_error_info is the "public" API where we should check the input - rdp_set_error_info on the other hand is an internal API (mostly from within the rdp context itself) so I think it's ok to assume a sane rdp input in that case.